tests/conftest.py

```diff
@@ -838,3 +846,11 @@ def cost_logger_fixture() -> Iterator[CostLogger]:
     print()
     print()
     print(cost_logger.log_cost_statistics())
+
+
+@pytest_asyncio.fixture(scope="function")
+async def simulation(consensus_mode, bt):
+    if consensus_mode != Mode.PLAIN:
+        pytest.skip("Skipping this run. This test only supports one running at a time.")
+    async for _ in setup_full_system(test_constants_modified, bt, db_version=1):
+        yield _
```

Comment:
why did you move this fixture from `test_simulation.py`?
and does `test_simulation_1()` also have the issue of only working with a single instance at a time?

Reply:
> why did you move this fixture from `test_simulation.py`?
Importing `Mode` from `conftest` would result in circular dependency.

> does `test_simulation_1()` also have the issue of only working with a single instance at a time?
Yes, I noticed that from the test runs.

Reply:
> Importing Mode from conftest would result in circular dependency.
it seems odd that it would be any different from importing `bt` from `conftest.py`
You typically don't have to explicitly import fixtures in order to use them

Reply:
You do when you `if consensus_mode != Mode.PLAIN:` though.

Reply:
I see